src/client/game.cpp

```diff
@@ -3893,9 +3893,9 @@ inline void Game::limitFps(FpsControl *fps_timings, f32 *dtime)
 	else
 		fps_timings->busy_time = 0;
 
-	u32 frametime_min = 1000 / (g_menumgr.pausesGame()
-			? g_settings->getFloat("pause_fps_max")
-			: g_settings->getFloat("fps_max"));
+	u32 frametime_min = 1000 / (device->isWindowFocused()
```

Comment:
Why is `g_menumgr.pausesGame()` no longer needed?

Reply:
I removed the pause_fps_max setting; I assume that it became fairly redundant because the framerate can now be limited when the game is unfocused.

Reply:
But the GUI might still allow lower frames, such as the pause menu. `pausesGame` should still be checked.